tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -161,9 +161,10 @@ Given('the user has opened folder {string}', (folder) => client.page.FilesPageEl
 When('the user opens folder {string} using the webUI', (folder) => client.page.FilesPageElement.filesList().navigateToFolder(folder))
 
 Given('the user has opened the share dialog for file/folder {string}', async function (fileName) {
-  await client.page.FilesPageElement.filesList().openSharingDialog(fileName)
-
-  return client
+  return client.page.FilesPageElement
+    .appSideBar()
+    .closeSidebar(100)
+    .openSharingDialog(fileName)
```

Comment:
why changing that? there is still a `openSharingDialog()` in the files list

Reply:
_so that we can chain different PO commands together_

but, this PR will be polluted with such changes..
reverted back to original..